### ISSUE TITLE
Requester should check that measurement indices are in ascending order.

### ISSUE THREAD

**Issue by steven-bellock** [DMTF/libspdm#1435]
>The measurement blocks in MeasurementRecord shall be sorted in ascending order by index.

**Comment by jyao1** [comment]
Same as https://github.com/DMTF/libspdm/issues/1434

**Comment by steven-bellock** [comment]
Yes, in the documentation we should be clear on what checks libspdm performs and which are left to the Integrator.
